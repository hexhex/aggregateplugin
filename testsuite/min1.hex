min(X) :- &min[p,mask](X).
count(X) :- &count[p,mask](X).
p(a).
p(b).
p(c).
q(e).
q(f).
g(u).
