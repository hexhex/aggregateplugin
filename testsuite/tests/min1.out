{g(u), q(f), q(e), p(c), p(b), p(a), min(a)}
